### ISSUE TITLE
Running tests in Docker

### ISSUE THREAD

**Issue by meguia-sfdc** [webdriverio-community/wdio-vscode-service#28]
Hello Team!

I'm trying to run wdio tests in a docker container and I can't make it work. The aim is to run wdio tests at a CI environment.

I start from a simple hello world project. Docker image starts from ubuntu:20.04 and I add some packages to compile and run tests. Nothing out of the standard stuff. We set Xvfb to run tests at a CI environment (Jenkins). I need java for other tasks.

```dockerfile
FROM ubuntu:20.04

ENV DEBIAN_FRONTEND noninteractive

RUN apt-get update

# Install required packages
RUN apt-get install --yes --no-install-recommends \
    dirmngr \ 
    gpg \
    build-essential \
    libgconf-2-4 \
    libgtk-3-0 \
    xvfb \
    libnss3-dev \
    libgbm-dev \
    libasound2 \
    xauth \
    awscli \
    npm \
    curl \
    openjdk-11-jdk-headless

# Install node
RUN curl -s https://deb.nodesource.com/setup_16.x | bash
RUN apt-get install nodejs -y
RUN node --version
RUN npm --version

# Setup screen
RUN Xvfb :99 -screen 0 1280x1024x24 -ac &
```

When running as a non root user, the I get this message:
```
2022-07-28T14:01:52.799Z INFO chromedriver: [FAKE VSCode Binary] STDERR: [812:0728/140152.797845:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/xxxxxxx/webdriver-test/.wdio-vscode-service/vscode-linux-x64-1.69.2/chrome-sandbox is owned by root and has mode 4755.
```

After applying the requested changes, I get the following:

```
2022-07-28T14:45:12.258Z INFO chromedriver: [FAKE VSCode Binary] STDERR: Failed to move to new namespace: PID namespaces supported, Network namespace supported, but failed: errno = Operation not permitted
```

Is there something missing? I see that your Github action works like charm.

Thank you!

**Comment by christian-bromann** [comment]
The GitHub Action environments are very well equipped and setup which is why running Chrome and at the end VS Code very easy. I don't know exactly what is missing but the error message suggests that VS Code isn't able to start inside the container.

**Comment by meguia-sfdc** [comment]
Do you mean the Github action as out of the box? Have you configured anything else by your own?

**Comment by christian-bromann** [comment]
> Do you mean the Github action as out of the box?

Yes. Maybe you can find the Docker container as a base somewhere in the GitHub org.

> Have you configured anything else by your own?

Nope. But happy to document this once we figured it out.

**Comment by meguia-sfdc** [comment]
Running container with `--privileged` and changing `chrome-sandbox` owner and permissions as requested works. Anyway, I don't think it's a good practice.

**Comment by meguia-sfdc** [comment]
I think the problem is that the "--no-sandbox" flag is not passed properly to fake vscode run command. Main process is sending "--sandbox=false". What do you think about changing
https://github.com/webdriverio-community/wdio-vscode-service/blob/main/src/chromium/index.ts#L25
by
```
configuration: { 'camel-case-expansion': true, 'boolean-negation': false }
```
? I could it make it run like this.

**Comment by christian-bromann** [comment]
Sounds good, mind raising a PR?

**Comment by christian-bromann** [comment]
Fixed in #30
